requirements.txt

```diff
@@ -4,7 +4,7 @@
 psycopg2==2.4.5  # requires libpq to build
 Django==1.7.1
 IPy==0.81
-twisted>=12,<13
+twisted>=12,<15
 simplejson==2.0.6  # freebsd doesn't have newer.
```

Comment:
Python 2.7 includes a json-library, same API as simplejson. Any reason we're not using that?